docs/src/FirstExample.md

```diff
@@ -185,6 +185,28 @@ end
 joint_model_chn = sample(example_joint_model(Y, t_m, T, Δ), NUTS(), 100)
 ```
 
+## Maximum Likelihood
+This implementation of joint models can also be used for maximum likelihood optimizations. The `Optim.jl` package contains many different optimizers that can be used to find parameters of your model. Here is an example of finding the parameters of the baseline hazard and link coefficient:
```

Comment:
Maybe need to say that this example is only estimating survival parameters, keeping longitudinal parameters fixed?

Reply:
named longitudinal parameters explicitly